### PR TITLE
localization fix

### PR DESCRIPTION
**Pull request by Clio18**
- create context and put there calling useTranslation() hook
- change calling useTranslation() hook to using it result from the context in each component

### CODE REVIEW

File: goods-partner-frontend/src/pages/Cars/CarFormDialog/CarFormDialog.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ interface CarFormDialogProps {
 }
 
 export const CarFormDialog: FC<CarFormDialogProps> = ({ title, closeDialog, open, actionHandler, car, setCar, drivers, t }) => {
-	// const { t } = useTranslation();
+	// const { t } = useContext(LanguageContext);
```
Comment (radeon323) [inline goods-partner-frontend/src/pages/Cars/CarFormDialog/CarFormDialog.tsx:25]: delete this one

File: goods-partner-frontend/src/components/ConfirmationDialog/ConfirmationDialog.tsx [hunk 2]
```diff
@@ -10,7 +11,8 @@ interface Props {
 }
 
 export const ConfirmationDialog = ({ open, setOpen, onAction, title, text }: Props) => {
-	const { t } = useTranslation();
+	// const { t } = useContext(LanguageContext);
```
Comment (radeon323) [inline goods-partner-frontend/src/components/ConfirmationDialog/ConfirmationDialog.tsx:14]: delete this one

File: goods-partner-frontend/src/pages/Delivery/DeliveryRoutes/RouteContent/RoutePointDetails/RoutePointDetailsBody/RoutePointDetailsBody.tsx [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
 import InfoTableItem from '../../../../../../components/InfoTableItem/InfoTableItem';
-import { RoutePoint } from '../../../../../../model/Route';
-import { useTranslation } from 'react-i18next';
+import { RoutePoint } from '@/model/Route';
+import { useContext } from 'react';
+import { LanguageContext } from '@/i18n/context';
 
 const RoutePointDetailsBody = ({ routePoint }: { routePoint: RoutePoint }) => {
-	const { t } = useTranslation();
+	//const { t } = useContext(LanguageContext);
```
Comment (radeon323) [inline goods-partner-frontend/src/pages/Delivery/DeliveryRoutes/RouteContent/RoutePointDetails/RoutePointDetailsBody/RoutePointDetailsBody.tsx:8]: delete this one

File: goods-partner-frontend/src/pages/Delivery/DeliveryRoutes/RouteContent/RoutePointDetails/RoutePointDetailsHeader/RoutePointDetailsHeader.tsx [hunk 4]
```diff
@@ -4,19 +4,20 @@ import RoutePointSelect from '../RoutePointSelect/RoutePointSelect';
...
+import { LanguageContext } from '@/i18n/context';
 
 interface RoutePointDetailsHeaderProps {
 	routePoint: RoutePoint;
 	routePointNumber: number;
 }
 
 const RoutePointDetailsHeader = ({ routePoint, routePointNumber }: RoutePointDetailsHeaderProps) => {
-	const { t } = useTranslation();
+	// const { t } = useContext(LanguageContext);
```
Comment (radeon323) [inline goods-partner-frontend/src/pages/Delivery/DeliveryRoutes/RouteContent/RoutePointDetails/RoutePointDetailsHeader/RoutePointDetailsHeader.tsx:19]: delete this one

**Review by radeon323 — CHANGES_REQUESTED** [review]

File: goods-partner-frontend/src/pages/Delivery/DeliveryRoutes/RouteContent/RoutePointDetails/RoutePointDetailsBody/RoutePointDetailsBody.tsx (hunk 3 above)
Comment (Clio18) [reply goods-partner-frontend/src/pages/Delivery/DeliveryRoutes/RouteContent/RoutePointDetails/RoutePointDetailsBody/RoutePointDetailsBody.tsx]: Done

File: goods-partner-frontend/src/pages/Delivery/DeliveryRoutes/RouteContent/RoutePointDetails/RoutePointDetailsHeader/RoutePointDetailsHeader.tsx (hunk 4 above)
Comment (Clio18) [reply goods-partner-frontend/src/pages/Delivery/DeliveryRoutes/RouteContent/RoutePointDetails/RoutePointDetailsHeader/RoutePointDetailsHeader.tsx]: Done

File: goods-partner-frontend/src/components/ConfirmationDialog/ConfirmationDialog.tsx (hunk 2 above)
Comment (Clio18) [reply goods-partner-frontend/src/components/ConfirmationDialog/ConfirmationDialog.tsx]: Done

File: goods-partner-frontend/src/pages/Cars/CarFormDialog/CarFormDialog.tsx (hunk 1 above)
Comment (Clio18) [reply goods-partner-frontend/src/pages/Cars/CarFormDialog/CarFormDialog.tsx]: Done

**Pull request merged**